### ISSUE TITLE
Found this randomly

### ISSUE THREAD

**Issue by BruhitsB** [Chocohead/Fabric-ASM#18]
Its not in my mods folder, and i cant use my worldedit commands now?? bro whats happening?????????

**Comment by ninjaguardian** [comment]
same

**Comment by zurgeg** [comment]
This is a library mod. There is a 99.9999% chance you are using optifabric, in which case, it's probably an incompatibility with optifabric. In which case: please read [this issue](https://github.com/modmuss50/OptiFabric/issues/242).
Optifabric requires this mod to patch Minecraft. If you are using Optifabric, please use [these more compatible alternatives](https://lambdaurora.dev/optifine_alternatives/).
On the 0.0001% chance you aren't, there is another mod that needs this, and it's interfering with worldedit.
**TL;DR**: If you are using OptiFabric, you should probably switch. If you aren't: one of your mods is interfering.
It's also possible you are coincidentally noticing Manningham Mills and this issue started happening at the same time. 
The most likely scenario is WorldEdit is broken. Reinstall it.
